docs/modules/deployment/pages/time-series-storage/timeseries/cortex.adoc

```diff
@@ -28,19 +28,23 @@ If your aim is to get up to speed quickly in a test environment, you can use Doc
 
 . Download the plugin's `.kar` file from GitHub (https://github.com/OpenNMS/opennms-cortex-tss-plugin/releases).
 . Copy the file into `$\{OPENNMS_HOME}/deploy`.
-. Enable and configure time series database plugins in {page-component-title}:
+. Enable and configure time series database plugins in `$\{OPENNMS_HOME}/etc/opennms.properties.d/timeseries.properties`:
```

Comment:
There is a ticket, NMS-15410 that says `opennms.properties.d` doesn't exist under `/examples`. Freddy is investigating for me. What am I missing? :-)

Reply:
It exists for me on 31.0.4.

Reply:
Ticket doesn't include "affects version" information. Probably don't need to worry about it here.